api/inventory/agents.proto

```diff
@@ -28,8 +28,8 @@ message PMMAgent {
 message NodeExporter {
   // Unique randomly generated instance identifier.
   string agent_id = 1;
-  // Node identifier where this instance runs.
-  string node_id = 2;
+  // pmm-agent identifier where this instance runs.
```

Comment:
```suggestion
  // The pmm-agent identifier which runs this instance.
```

Do the same everywhere.

Reply:
This is your comment just copied from https://github.com/percona/pmm/blob/0efdbf9d6a2a97301af3849456dacbd0189839be/api/inventory/agents.proto#L151 but nevermind, I'll fix that.

Reply:
Yes, but it makes less sense. :) "where" points to the place. The place stays the same – some Node (host). pmm-agent is not a place – it is a way to run agent.